### PR TITLE
removed Rust and Cargo Audit, edited Dockerfile to use custom image, …

### PR DESCRIPTION
**Pull request by courtney-weglar**
…removed external image references from orb.yml

### CODE REVIEW

**Comment by jbornemann** [comment]
[check]

**Comment by pendo-ops** [comment]
Okay, All checks were run. Did not encounter any errors.

**Comment by jbornemann** [comment]
[check]

**Comment by pendo-ops** [comment]
There was an error adding Reviewers to the Issue PRODSEC-227: The issue does not exist in Jira.

**Comment by pendo-ops** [comment]
Okay, All checks were run. Did not encounter any errors.

**Review by jbornemann — APPROVED** [review]
I dont' see anything that concerns me, but I don't really have the background on why we are removing rust functionality vs others or why remove at all.

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,10 +9,17 @@ jobs:
       - run: docker run salus-tests
       - run: bin/integration_test
   salus:
-    machine: true
+    machine:
+      image: ubuntu-1604:201903-01
```
Comment (moor1204) [inline .circleci/config.yml:13]: Is this the image that is universally used for Docker?

**Review by moor1204 — APPROVED** [review]
One non-blocking comment. We may have to keep an eye on this in the future, as it is an older version of Ubuntu. This is approved.

**Comment by pendo-ops** [comment]
All JIRA issue transitioned/labeled.
* [DEVOPS-2161](https://pendo-io.atlassian.net/browse/DEVOPS-2161): Jira Issue DEVOPS-2161: Changed to Resolved


Branch PRODSEC-227-salus-integration deleted

**Pull request merged**